zendesk/tag.go

```diff
@@ -40,6 +42,76 @@ func (z *Client) GetTicketTags(ctx context.Context, ticketID int64) ([]Tag, erro
 	return result.Tags, err
 }
 
+type TagListOptions struct {
+	PageOptions
+}
+
+// ListTags gets list of available tags
+//
+// ref: https://developer.zendesk.com/api-reference/ticketing/ticket-management/tags/#list-tags
+func (z *Client) ListTags(ctx context.Context, options TagListOptions) ([]Tag, Page, error) {
+	var result struct {
+		Page
+		Tags []struct {
+			Name  string `json:"name"`
+			Count int64  `json:"count"`
+		} `json:"tags"`
+	}
+
+	u, err := addOptions("/tags.json", options)
+	if err != nil {
+		return nil, Page{}, err
+	}
+
+	body, err := z.get(ctx, u)
+	if err != nil {
+		return nil, Page{}, err
+	}
+
+	err = json.Unmarshal(body, &result)
+	if err != nil {
+		return nil, Page{}, err
+	}
+
+	var tags []Tag
+	for _, t := range result.Tags {
+		tags = append(tags, Tag(t.Name))
+	}
+
+	return tags, result.Page, nil
+}
+
+type SearchTagsOptions struct {
+	PageOptions
+	Query string `url:"name,omitempty"`
```

Comment:
```suggestion
	Name string `url:"name,omitempty"`
```

just to avoid confusion, see other examples - https://github.com/watchtowerai/go-zendesk/blob/master/zendesk/group_membership.go#L27

Reply:
fixed ✅